.github/ISSUE_TEMPLATE/bug_feedback.yml

```
@@ -0,0 +1,62 @@
+name: '🐛 BUG反馈'
+description: 向我们反馈你遇到的错误
+title: "[BUG]: "
+labels: ["bug", "DEV-需要注意", "DEV-未经测试"]
+body:
+  - type: checkboxes
+    attributes:
+      label: 在继续之前，请确认以下内容
+      description: 在发布您的问题之前，请确认以下内容。
+      options:
+        - label: 我已经搜索过已存在的问题，并没有存在相同的正在进行中/重复/已解决的问题
+          required: true
+        - label: 我知道由于作者精力和知识有限，可能不会处理一些错误
+          required: true
+  - type: dropdown
+    attributes:
+      label: 选择一个问题类别
+      description: 你遇到了什么样的问题?
+      options:
+        - 文档错误
+        - 显示错误
+        - 功能错误
+        - 其他
```

```suggestion
        - 使用错误
        - 显示错误
        - API错误
        - 其他
```

文档相关问题不应该使用 https://github.com/DuckDuckStudio/GitHub-Labels-Manager/labels/bug 标签